suggerimenti-di-scrittura/punteggiatura-e-grammatica.rst

```diff
@@ -15,10 +39,12 @@ L’USO DEL MAIUSCOLO PER INTERE FRASI È DA EVITARE: RENDE PIÙ DIFFICILE LA LE
 
 In generale usa iniziali minuscole, tranne quando si tratta di nomi propri o di eccezioni come:
 
--  titoli istituzionali completi, seguiti dal nome proprio (il Ministro per le politiche agricole Nome Cognome);
+-  titoli istituzionali completi, seguiti dal nome proprio (il Ministro per le Politiche agricole Nome Cognome);
```

Comment:
Questo credo sia da lasciare in minuscolo:

* https://it.wikipedia.org/wiki/Ministero_delle_politiche_agricole_alimentari_e_forestali
* https://docs.italia.it/italia/designers-italia/writing-toolkit/it/bozza/le-parole-della-pubblica-amministrazione/m.html?highlight=ministero

Reply:
Si può trovare molta disomogeneità sul web (e non solo): è diffuso sia l'uso di tutte le prime lettere in maiuscolo (https://www.mef.gov.it/index.html, https://www.mite.gov.it/, https://www.esteri.it/it/), che quello di tutte le lettere in minuscolo (https://www.mise.gov.it/index.php/it/). 
L'iniziale degli organi (es. Ministero, Consiglio ecc.) è di norma maiuscola, minuscola se usata in modo generico (Manuale di stile). Anche l'iniziale della prima parola che individua l'ambito di competenza (Ministero della Difesa, Ministero della Salute ecc.) è maiuscola, questo consente (senza abusare delle maiuscole come richiesto già dalla Direttiva Frattini) di articolare un elenco formato da più ministeri, distinguendo così dove finisce una e comincia l'altra e di agevolare la lettura dei testi (guidando l'occhio) soprattutto quando i nomi dei ministeri sono molto articolati. Ad es. I Ministeri dell'Ambiente e della tutela del territorio e del mare, del Lavoro e delle politiche sociali e delle Infrastrutture e della mobilità... In termini di accessibilità, inoltre, questa scelta si presenta come un valore aggiunto, basta pensare alle persone con difficoltà di apprendimento e disabilità cognitive.  Al riguardo avrei preparato anche le modifiche alla lettera M per ministro e ministero (sarebbe in una delle prossime pull). Che ne pensi?